packages/db/postgres/tables.sql

```diff
@@ -48,34 +48,53 @@ CREATE INDEX IF NOT EXISTS content_updated_at_idx ON content (updated_at);
 -- IPFS Cluster tracker status values.
 -- https://github.com/ipfs/ipfs-cluster/blob/54c3608899754412861e69ee81ca8f676f7e294b/api/types.go#L52-L83
 -- TODO: nft.storage only using a subset of these: https://github.com/ipfs-shipyard/nft.storage/blob/main/packages/api/db/tables.sql#L2-L7
-CREATE TYPE pin_status_type AS ENUM
-(
-  -- Should never see this value. When used as a filter. It means "all".
-  'Undefined',
-  -- The cluster node is offline or not responding.
-  'ClusterError',
-  -- An error occurred pinning.
-  'PinError',
-  -- An error occurred unpinning.
-  'UnpinError',
-  -- The IPFS daemon has pinned the item.
-  'Pinned',
-  -- The IPFS daemon is currently pinning the item.
-  'Pinning',
-  -- The IPFS daemon is currently unpinning the item.
-  'Unpinning',
-  -- The IPFS daemon is not pinning the item.
-  'Unpinned',
-  -- The IPFS daemon is not pinning the item but it is being tracked.
-  'Remote',
-  -- The item has been queued for pinning on the IPFS daemon.
-  'PinQueued',
-  -- The item has been queued for unpinning on the IPFS daemon.
-  'UnpinQueued',
-  -- The IPFS daemon is not pinning the item through this CID but it is tracked
-  -- in a cluster dag
-  'Sharded'
-);
+DO $$
+BEGIN
```

Comment:
Do we need this outer `DO` block?

Reply:
As far as I understand it, IF statements are part of pgSQL.
So in order to be used they need to be part of a function or block statement, hence the use of a `DO` block.

I'm not an expert in pgSQL though, so really happy to be proven wrong and be advised on better ways to achieve this.
Do you have anything in mind?

Reply:
I tried other alternatives but I think that is as @flea89 says and we must use the DO block